packages/services/src/__tests__/aggregationEvent.test.ts

```diff
@@ -59,7 +64,8 @@ describe('processAggregationEvent', () => {
   it('should process aggregation event', async () => {
     (vckitService.issueVC as jest.Mock).mockImplementationOnce(() => aggregationVCMock);
     (vckitService.decodeEnvelopedVC as jest.Mock).mockReturnValue(aggregationVCMock);
-    (uploadData as jest.Mock).mockResolvedValueOnce(aggregationEventDLRMock);
+    (uploadData as jest.Mock).mockResolvedValueOnce({ uri: 'https://exampleStorage.com/vc.json', key: '123', hash: 'ABC123' });
+    (constructVerifyURL as jest.Mock).mockReturnValueOnce('http://localhost/event/1234');
```

Comment:
Hi, should we change the link return to follow the `verifyURL` return?
<img width="582" alt="image" src="https://github.com/user-attachments/assets/1f096671-c5ab-4acf-b4fc-7bfb1ef30da5">

Reply:
I understand you want the `constructVerifyURL` return the structure string contains the `/verify` string there, but I don't think we need to do that, since the `constructVerifyURL` function is a side effect function, we not test it in this test case, and the response from that function in this case that not valuable, you can see some lines below this line, where the data is respond by `constructVerifyURL` function is used by `registerLinkResolver` function, which already mocked. 

I understand that you want the `constructVerifyURL` function to return a structured string containing `/verify`, but I don’t think this is necessary. Since `constructVerifyURL` is a side effect function, we aren’t testing it directly in this case, and its response isn’t relevant here. Just a few lines below, you can see that the data returned by `constructVerifyURL` is used by the `registerLinkResolver` function, which is already mocked.